src/java-runtime/index.ts

```diff
@@ -9,6 +9,51 @@ import * as pathExists from "path-exists";
 import * as request from "request-promise-native";
 import findJavaHome = require("find-java-home");
 import architecture = require("arch");
+import { loadTextFromFile, getExtensionContext } from "../utils";
+
+let javaRuntimeView: vscode.WebviewPanel | undefined;
+
+export async function javaRuntimeCmdHandler(context: vscode.ExtensionContext, operationId: string) {
+  if (javaRuntimeView) {
+    javaRuntimeView.reveal();
+    return;
+  }
+
+  javaRuntimeView = vscode.window.createWebviewPanel("java.runtime", "Java Runtime", {
+    viewColumn: vscode.ViewColumn.One,
+  }, {
+    enableScripts: true,
+    enableCommandUris: true,
+    retainContextWhenHidden: true
+  });
+
+  await initializeJavaRuntimeView(context, javaRuntimeView, onDidDisposeWebviewPanel);
+}
+
+function onDidDisposeWebviewPanel() {
+  javaRuntimeView = undefined;
```

Comment:
Shall we call `javaRuntimeView.dispose()` before set it to `undefined`?

Reply:
This event handler is called when the dispose() is called. So there's no need to call it again.